pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/AdditionTransformFunctionTest.java

```diff
@@ -69,8 +69,8 @@ public void testAdditionTransformFunction() {
     testTransformFunction(transformFunction, expectedBigDecimalValues);
 
     expression = RequestContextUtils.getExpression(
-        String.format("add(add(12,%s),%s,add(add(%s,%s),'12110.34556677889901122335678',%s),%s)", STRING_SV_COLUMN,
```

Comment:
Does it mean we won't auto infer big decimal?

Reply:
Correct. In standard SQL, literal cannot be auto inferred based on its value.